wiki/Interface/ko.md

```diff
@@ -243,7 +243,7 @@
 
 `Mods` 버튼을 누르거나 `F1` 키를 누르면 **[게임 내 모드 선택](/wiki/Game_Modifiers)** 창이 나타납니다.
 
-이 화면에서는 게임 내 모드 (줄여서 `모드`)를 선택할 수 있습니다. 이러한 모드는 게임의 난이도를 낮추지만 얻은 점수의 배율을 낮추기도 하고, 게임의 난이도를 높이지만 얻은 점수의 배율을 높이기도 합니다. 어떤 모드는 완전히 다른 방식으로 게임을 바꾸기도 합니다. [Relax](/wiki/Game_Modifiers/#Relax)와 [Auto Pilot](/wiki/Game_Modifiers/#Auto_Pilot)이 이러한 모드에 속합니다.
+이 화면에서는 게임 내 모드 (줄여서 `모드`)를 선택할 수 있습니다. 이러한 모드는 게임의 난이도를 낮추지만 얻은 점수의 배율을 낮추기도 하고, 게임의 난이도를 높이지만 얻은 점수의 배율을 높이기도 합니다. 어떤 모드는 완전히 다른 방식으로 게임을 바꾸기도 합니다. [Relax](/wiki/Game_Modifiers#Relax)와 [Auto Pilot](/wiki/Game_Modifiers#Auto_Pilot)이 이러한 모드에 속합니다.
```

Comment:
broken anchor

Reply:
that's one of the things I put in PR description cuz I'll do it later